File: backend/src/main/java/com/tododuk/domain/todoLabel/entity/TodoLabel.java
package com.tododuk.domain.todoLabel.entity;

import com.tododuk.domain.label.entity.Label;
import com.tododuk.domain.todo.entity.Todo;
import com.tododuk.global.entity.BaseEntity;
import jakarta.persistence.Entity;
import jakarta.persistence.ManyToOne;
import lombok.AllArgsConstructor;
import lombok.Builder;
import lombok.Getter;
import lombok.NoArgsConstructor;

@Getter
@NoArgsConstructor
@Entity
@Builder
@AllArgsConstructor
public class TodoLabel extends BaseEntity {

    @ManyToOne
    private Todo todo;

    @ManyToOne
    private Label label;
}

File: backend/src/main/java/com/tododuk/domain/todoLabel/entity/TodoLabel.kt
package com.tododuk.domain.todoLabel.entity

import com.tododuk.domain.label.entity.Label
import com.tododuk.domain.todo.entity.Todo
import com.tododuk.global.entity.BaseEntity
import jakarta.persistence.Entity
import jakarta.persistence.FetchType
import jakarta.persistence.ManyToOne


@Entity
class TodoLabel(
    @ManyToOne(fetch = FetchType.LAZY)
    var todo: Todo? = null,

    @ManyToOne(fetch = FetchType.LAZY)
    var label: Label? = null
) : BaseEntity(){

}
